app/src/main/java/com/android/wardrobeManager/ui/images/ClothingItemImageManager.java

```diff
@@ -104,7 +104,7 @@ private static Bitmap generateClothingItemImage(ClothingItem toLoad) {
         Bitmap reference = Bitmap.createScaledBitmap(drawableToBitmap(d), bitmap.getWidth(), bitmap.getHeight(), false);
 
         int[] colors = Utility.parseClothingItemColors(toLoad.getColors());
-        return ColorStyleFilterManager.filterColorStyle(toLoad.getDesign(), bitmap, reference, colors);
+        return DesignFilterManager.filterColorStyle(toLoad.getDesign(), bitmap, reference, colors);
```

Comment:
I see you changed this from ColorStyle to Design which is great, but you missed another ColorStyle on this line. There are also several other references using the phrase "ColorStyle" throughout the rest of the codebase. You can find them with Ctrl+shift+f. Just be sure to replace them with design at some point.